test/uritests.txt

```diff
@@ -0,0 +1,2211 @@
+uri     = http://user@f.q.d.n:8080/foo/bar/cookie.crt?n=2#foo
+scheme  = http
+authority = user@f.q.d.n:8080
+path    = /foo/bar/cookie.crt
+query   = n=2
+fragment = foo
+user    = user
+host    = f.q.d.n
+service = 8080
+
+uri     = file:foo.pem?n=1
+scheme  = file
+path    = foo.pem
+query   = n=1
+
+uri     = bar.pem
+path    = bar.pem
+
+######################################################################
+#
+# Shamelessly stolen from
+# https://github.com/cweb/iri-tests/blob/master/iris.txt
```

Comment:
It's an MIT license, I hope you find it useful.  You may want to use urls.json instead though, if you can deal with this format:  https://github.com/cweb/url-testing/blob/master/urls.json  It's an updated version of iris.json, which iris.txt derives from.